src/main/java/com/github/manolo8/darkbot/backpage/auction/AuctionItems.java

```diff
@@ -92,10 +91,4 @@ public String toString() {
                 "instant=" + instantBuy +
                 "}";
     }
-
-    protected enum AuctionType {
-        HOUR,
-        DAY,
-        WEEK
-    }
```

Comment:
Don't get rid of AuctionType, keep it, and when sending the request just have a method that turns it into the string you want to send (just add a `String getId()` implemented as `name().toLowerCase(Locale.ROOT)`

Also, because you have this inside AuctionItems, you probably can just name it Type, so it ends up being AuctionItems.Type